src/main/java/io/codelex/collections/practice/Histogram.java

```diff
@@ -6,20 +6,59 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.util.*;
+
 
 public class Histogram {
```

Comment:
Šajā klasē izskatās, ka ir samiksētas statiskas un ne-statiskas lietas. Jebšu klaše Histogram ir main metode, kurā taisi jaunu objektu no Histogram klases.
Vienkāršākais risinājums, kā to izlabot - visu saliec statisku un pārveido main metodi:
```
public static void main(String[] args) throws IOException {
        generateHistogram();
        printHistogram();
    }
```
Un toString metodi pārdēvē par printHistogram, arī liec statisku un ņem nost @Override anotāciju